VimeoUpload/Upload/Controllers/VideoDeletionManager.swift

```diff
@@ -131,28 +134,29 @@ public class VideoDeletionManager: NSObject
     
     // MARK: Archiving
     
-    private func loadDeletions() -> [VideoUri: Int]?
+    private func loadDeletions(withMigrator migrator: ArchiveMigrating?) -> [VideoUri: Int]
     {
         guard self.shouldLoadArchive == true else
         {
-            return nil
+            return [:]
         }
         
-        if let deletions = self.archiver.loadObject(for: type(of: self).DeletionsArchiveKey) as? [VideoUri: Int]
+        let relativeFolderPath = VideoDeletionManager.DeletionsArchiveKey + "/" + VideoDeletionManager.DeletionsArchiveKey
+        
+        guard let retries = ArchiveDataLoader.loadData(relativeFolderPath: relativeFolderPath,
```

Comment:
Are retries stored in the deletions archive?

I also thought it was kind of funny that the directory structure was `path/to/here/deletions/deletions`, but I think that's legacy stuff, right?

Reply:
I think Alfie made a mistake when he wrote this class lol. And yes, from my investigation, the number that we tag with the video URI is the number of retries for a specific video.